### ISSUE TITLE
How many training steps used to obtain the pre-trained model? 

### ISSUE THREAD

**Issue by QuXinghuaNTU** [araffin/rl-baselines-zoo#70]
Is there any document illustrating how many training steps used to obtain the pre-trained model? Some pretrained model seems far less than the start-of-the-art. For instance, the dqn model on BeamRider and Qbert only achieve 948.0 and 550.0. However, using other policies (e.g., PPO2 and ACKTR), such reward values could be 10,000+.
It would be better if you can provide these pre-trained models as a trustworthy baseline for benchmarking.

**Comment by araffin** [comment]
duplicate of #38
